plugin/editorconfig.vim

```diff
@@ -503,18 +507,20 @@ function! s:ApplyConfig(config) " {{{1
 
 " Set the indentation style according to the config values
 
-    if has_key(a:config, "indent_style")
+    if index(g:EditorConfig_disable_rules, 'indent_style') < 0 &&
+                \ has_key(a:config, "indent_style")
```

Comment:
sorry, i'm really newbie in vimscript (and english), i didn't get which form of function do you suppose. do you want me to add smth like
```
function isRuleActive(name)
  return index(g:EditorConfig_disable_rules, a:name) < 0 &&
                 \ has_key(a:config, a:name) 
endfunction
```
?